.github/workflows/maturin_upload_pypi.yml

```diff
@@ -34,9 +34,9 @@ jobs:
           manylinux: auto
           working-directory: lib/
       - name: Upload wheels
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
-          name: wheels
+          name: wheels-linux-${{ matrix.target }}
```

Comment:
Why this changed?

Reply:
Seems like this is a maturin thing. It may be better to just regenerate this with the command at the top of the file